internal/config/version/version.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package version
+
+const (
+	Bootloader Type = "bootloader" // Bootloader is the version of the Azure SEVSNP bootloader.
+	TEE        Type = "tee"        // TEE is the version of the Azure SEVSNP TEE.
+	SNP        Type = "snp"        // SNP is the version of the Azure SEVSNP SNP.
+	Microcode  Type = "microcode"  // Microcode is the version of the Azure SEVSNP microcode.
+)
+
+// Type is the type of the version to be requested.
+type Type (string)
+
+// GetVersion returns the version of the given type.
+func GetVersion(t Type) uint8 {
+	switch t {
+	case Bootloader:
+		return 2
+	case TEE:
+		return 0
+	case SNP:
+		return 6
+	case Microcode:
+		return 93
+	default:
+		return 1
```

Comment:
Not sure what's a good default here (if any). May also depend on how the caller uses the value. For the current use case, MaxUint8 may be the secure choice so that the attestation doesn't accidentally accept an outdated version.
One could also argue that this is a "can't happen"/logic error and panic.
Or return an error, which may be needed anyway when getting the value from the API.
(maybe @daniel-weisse also has an opinion on this)

Reply:
I final version of the code which loads these values from our API will probably look a bit different.
IMO throwing a panic here works to make sure we dont have any logic errors in the config code

Reply:
I agree, panicking makes even sense. I think failing early is a good principle. And also, the function will be replaced with the S3 logic in the next PR :) 
Would appreciate some design feedback here too https://github.com/edgelesssys/constellation/pull/1808